htmlassignment/contactme.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+   <!-- <meta http-equiv="X-UA-Compatible" content="IE=edge" />-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Assignment</title>
+    <link href="../css/assignment.css" rel="stylesheet" />
+  </head>
+  <body>
+    <nav>
+      <label class="logo"><marquee>VIDYA  JYOTHI  INSTITUTE  OF  TECHNOLOGY </marquee></label>
+      <ul>
+        <li><a  href="../htmlassignment/index.html">Home</a></li>
+        <li><a  href="../htmlassignment/aboutme.html">About Me</a></li>
+        <li><a  class="active" href="../htmlassignment/contactme.html">Contact</a></li>
+        <li><a href="../htmlassignment/gallery.html">Gallery</a></li>
+      </ul>
+    </nav>
+    <section>
+      
+      
+     </section>
+     <h1><center>DETAILS</center></h1><br> 
+     <div class="contactsdetaails">
+      <div class="contactform">
+        <form id="form">
+          <br><input type="text" placeholder="enter your name" ><br><br>
+             <input type="number" placeholder="enter your number" ><br><br>
+             <input type="email" placeholder="enter your email(@gmail.com)" ><br><br>
+             <input type="textarea" placeholder="write your problem" ><br><br>
+             <button type="submit" ><center> SUBMIT</center></button>
+        </form>
+      </div>
+      
+      <div class="contactform1">
+        <div><h1><b>Address</b></h1>
+          </div><br>
+            Vidya Jyothi Institute of Technology
+            Aziz Nagar Gate C.B. Post
+            Hyderabad–500 075
+            Telangana, India
+            <br><br>
+            
+            Phone
+            73 73 637 637
+            91 – 8413 – 235300
+            91 – 8413 -235399
+            Extension Numbers
+            <br><br>
+            Email
+            info@vjit.ac.in
+        </div>
+      
+      <!-- <div class="formBox">
```

Comment:
Remove the commented lines if you don't use them

Reply:
commented lines are removed